Rocket.Chat/Models/Attachment.swift

```diff
@@ -29,6 +29,7 @@ class Attachment: BaseModel {
 
     @objc dynamic var collapsed: Bool = false
     @objc dynamic var text: String?
+    @objc dynamic var desc: String?
```

Comment:
Could you change it to `descriptionText` please? Just to follow the same pattern we have in other models.